README.md

```diff
@@ -38,7 +38,7 @@ const data = [[1, 2, 3], [true, false, null, 'sheetjs'], ['foo', 'bar', new Date
 var buffer = xlsx.build([{name: "mySheetName", data: data}]); // Returns a buffer
 ```
 
-  * Building a xlsx (spannig multiple rows `A1:A4`)
+  * Building a xlsx (spannig multiple rows `A1:A4` in all sheets)
```

Comment:
"in all sheets" doesn't seem to be correct description.
As you changes, global merge option will be applied only given one sheet.

Reply:
Done